js/src/contracts/registry.js

```diff
@@ -101,4 +101,17 @@ export default class Registry {
       return address;
     });
   }
+
+  lookupMeta (_name, key) {
```

Comment:
I had it that way around initially since it is 99.9% shared. Not sure why it was reverted, will take a re-look.